int32 node_id
int32 target
---
float64 chosen_bearing
